src/pages/ReportDetailsPage.tsx

```diff
@@ -416,12 +417,23 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 action: () => unapproveExpenseReportOrShowModal(),
             });
         }
+
```

Comment:
```suggestion
const isDebugModeEnabled = !!user?.isDebugModeEnabled
```

Should create a new variable